Sources/News.php

```diff
@@ -644,14 +767,18 @@ function getXmlMembers($xml_format)
 		SELECT id_member, member_name, real_name, date_registered, last_login
 		FROM {db_prefix}members
 		ORDER BY id_member DESC
-		LIMIT {int:limit}',
+		LIMIT {int:limit} OFFSET {int:offset}',
```

Comment:
That you use offset i find nice (from sql style pov),
but i would like to point out that i used because i had to use it in a sub query.
Normaly the query api of smf add the word offset by his own when needed.